test/src/specs/sync/get_blocks.rs

```diff
@@ -53,14 +53,14 @@ impl Spec for GetBlocksTimeout {
 
         let received = wait_get_blocks_point(&net, &node1, block_download_timeout_secs * 2, 1);
         assert!(
-            received.is_none(),
-            "Should not received GetBlocks anymore, the timeout could be any number."
+            received.is_some(),
```

Comment:
we shouldn't modify the assertion here, please setup up more dummy nodes and connect them to node1.

Reply:
Here, our test purpose has changed. The current test is that even if the timeout is triggered, the eviction will not occur if the number of connections is not enough. 

---

By the way, I think it is currently impossible to measure eviction scenarios in multiple node states. The reason is as follows：
1. There is no problem with the generation of multiple connections, but multiple connections are processing downloads at the same time, and this is considered uncertain 
2. Even if multiple connections are processing download tasks at the same time, but you need to respond to random download tasks on other connections, and do not respond on a specific node, this is not a well-controlled process, because the generation of download tasks is not caused by you control, but the node started by another process

Reply:
the assertion message of line 57 and line 66 should be modified accordingly if you want to change the test spec, "Should received GetBlocks anymore" looks weird...

Reply:
I rewrote the panic message, please review again